src/schemas/model-data-schema.json

```
@@ -153,67 +109,125 @@
                     "type":"string"
                 },
                 "owner":{
-                    "oneOf":[
-                        {"$ref":"#/definitions/uid"},
-                        {"$ref":"#/definitions/entity"}
-                    ]
+                    "$ref":"#/definitions/guid"
                 },
                 "ownee":{
-                    "oneOf":[
-                        {"$ref":"#/definitions/uid"},
-                        {"$ref":"#/definitions/entity"}
-                    ]
+                    "$ref":"#/definitions/guid"
                 }
             },
             "required":[
+                "guid",
                 "name",
                 "description",
                 "owner",
                 "ownee"
             ]
         },
-        "database": {
+        "multiplicity":{
+            "type":"object",
+            "properties": {
+                "lowerBound":{
+                    "$ref": "#/definitions/multiplicityBoundary"
+                },
+                "UpperBound":{
```

Still have typo here
```suggestion
                "upperBound":{
```